.gitignore

```diff
@@ -1,12 +1,16 @@
 .vscode/
+.vs/*
```

Comment:
Is this from a fork? This shouldn't be pushed upstream.

Reply:
fork